scripts/devaccess/devaccess_wrap.py

```diff
@@ -270,19 +304,51 @@ def rpm_qa(cmd):
         return results
 
     @staticmethod
-    def oc_get_nodes(occmd):
-        ''' provide 'oc get nodes' '''
+    def cmd_with_tail(cmd):
+        ''' replace process with called one to keep getting updates '''
+        args = cmd.split()
+        os.execvp(args[0], args)
 
-        n_cmd = occmd.normalized_cmd()
-        run_cmd = WhitelistedCommands.oc_cmd_builder(n_cmd)
-        results = subprocess.check_output(run_cmd)
+    @staticmethod
+    def safe_command(cmd):
+        ''' return any safe command output
+            this is for all commands that do not have any type of sensitive data in them
```

Comment:
This is more aimed at future enhancements, but safe vs non-safe also applies to read or read-write type commands, based on arguments and user.  e.g. "date" can set system clock and break a system if run as root, but usual use is read-only and harmless.